templates/main.cf.j2

```diff
@@ -1,4 +1,7 @@
 # {{ ansible_managed }}
+{% if postfix_custom_main is defined %}
+{{ postfix_main }}
```

Comment:
I guess you also want to write {{ postfix_custom_main }} here?

Reply:
But of course, sorry about the typo. I originally had just postfix_main, but thought that postfix_custom_main would be a better name for the variable in the context.

Reply:
I agree that postfix_custom_main is better :)